Library/Homebrew/bintray.rb

```diff
@@ -59,19 +68,25 @@ def upload(local_file, repo:, package:, version:, remote_file:, sha256: nil, war
     args = ["--upload-file", local_file]
     args += ["--header", "X-Checksum-Sha2: #{sha256}"] unless sha256.blank?
     args << "--fail" unless warn_on_error
-    result = open_api(url, *args)
+
+    result = open_api(url, T.unsafe(*T.unsafe(args)))
```

Comment:
This was recommended to me on Sorbet's Slack:

```suggestion
    result = T.unsafe(self).open_api(url, *args)
```

Reply:
`T.unsafe` should be around the receiver (`self`), since the whole call is unsafe in this case, not just the splat.